tests/integration/static_ip_address_test.py

```diff
@@ -57,3 +76,46 @@ def test_add_static_ipv4_with_min_state():
     netapplier.apply(copy.deepcopy(desired_state))
 
     assertlib.assert_state(desired_state)
+
+
+def test_remove_static_ipv4(setup_eth1_ipv4):
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'eth1',
+                'type': 'ethernet',
+                'ipv4': {
+                    'enabled': False
+                }
+            }
+        ]
+    }
+
+    print desired_state
+    netapplier.apply(copy.deepcopy(desired_state))
+
+    assertlib.assert_state(desired_state)
+
+
+
+
+def test_edit_static_ipv4_address_and_prefix(setup_eth1_ipv4):
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'eth1',
+                'type': 'ethernet',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'address': [
+                        {'ip': '192.168.122.64', 'prefix-length': 23}
```

Comment:
Please use a designated test address, such as 192.0.2.251, see 
https://tools.ietf.org/html/rfc5737#section-3 - this will reduce problems with address conflicts.